CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt-utils v0.6.3
+
+## Features
```

Comment:
ok so this is on us — we changed how we're using branches in dbt-utils and forgot to update the PR template.

I've just switched the base branch for this to be `dev/0.7.0` — can you rebase this work from that branch, and add this entry into the `dev/0.7.0` changelog section?